src/cmds/scripts/pbs_topologyinfo.py

```diff
@@ -254,12 +255,15 @@ def socketXMLstart(name, attrs):
         if (name == "object" and attrs.get("type") == "OSDev" and
             attrs.get("osdev_type") == "1" and
                 attrs.get("name").startswith("card")):
-            inventory.gpudevices += 1
-        if (name == "object" and attrs.get("type") == "OSDev" and
-            attrs.get("osdev_type") == "1" and
-                attrs.get("name").startswith("controlD")):
-            if (inventory.gpudevices > 0):
-                inventory.gpudevices -= 1
+            inventory.flagc = True
+        elif (name == "object" and attrs.get("type") == "OSDev" and
+              attrs.get("osdev_type") == "1" and
+                attrs.get("name").startswith("renderD")):
+            if inventory.flagc is True:
+                inventory.gpudevices += 1
+                inventory.flagc = False
+        else:
+            inventory.flagc = False
```

Comment:
If you are using class variable here, why can't the same be used nstead of local variables in the previous function?